Libraries/Image/RCTImageView.mm

```diff
@@ -47,7 +47,9 @@ static BOOL RCTShouldReloadImageForSizeChange(CGSize currentSize, CGSize idealSi
   }
 
   NSSize originalImageSize = originalImage.size;
-  if (NSEqualSizes(originalImageSize, NSZeroSize) || [[originalImage representations] count] == 0) {
+  if (NSEqualSizes(originalImageSize, NSZeroSize) ||
+      NSEqualSizes(originalImageSize, targetSize) ||
```

Comment:
Does this change exist upstream? (On my phone.)

Reply:
This function is macOS only, from the comment above:
> Implements macOS equivalent behavior of UIViewContentModeScaleAspectFill.
> Used for RCTResizeModeCover support.

Reply:
Got it 👍